Gemfile

```diff
@@ -15,6 +15,8 @@ gem "puma", "~> 6.4"
 gem "rails", "~> 7.1.3.4"
 gem "redis"
 gem "sidekiq"
+gem "sidekiq-throttled", '1.4.0' # '1.5.0' was losing some jobs
```

Comment:
This is quite an ominous comment :smile:

Reply:
Yes, but the gem will be not needed after we migrate to Solid Queue.